src/test/java/org/elasticsearch/test/store/MockFSDirectoryService.java

```diff
@@ -92,7 +88,9 @@ public void beforeIndexShardClosed(ShardId sid, @Nullable IndexShard indexShard,
                             // By doing a commit flush we perform a Lucene commit, but don't clear the translog,
                             // so that even in tests where don't flush we can check the integrity of the Lucene index
                             logger.info("{} flushing in order to run checkindex", indexShard.shardId());
```

Comment:
should the start/finish flush messages be inside the if block? seems like we would only want to see them when a flush actually happens